pom.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.airlift</groupId>
+        <artifactId>airbase</artifactId>
+        <version>199</version>
+    </parent>
+
+    <artifactId>launcher-v2</artifactId>
```

Comment:
launcher

Reply:
This will conflict with airlift launcher (same GAV)

Reply:
But the released versions are still there.

Reply:
We can start with a higher release number so that this one is just a continuation of the older one.

Reply:
One higher than the last version of the old launcher that gets published.

Reply:
Airlift is now 285, let's start from version 300. I've updated GAV